sepsis-quiz-js/src/sepsisQuiz.js

```diff
@@ -97,11 +105,6 @@ class SepsisQuiz {
       answer = answer ? allOfTheAbove : noneOfTheAbove
       choices = SepsisQuiz.shuffleArray(wrongAnswers).concat(answer)
     }
-    // when the one of the incorrect answers is "all of the  above" or "none of the above"
```

Comment:
did this cause one of the bugs?  It would be nice to have this to future proof this for them